### PR TITLE
[SQL-Gen] Support Blob type in Postgres

### PR DESCRIPTION
**Pull request by jaylees14**
- Add support for Blob in Postgres: [see docs](https://www.postgresql.org/docs/9.1/datatype-binary.html)
- Add unit and integration test to validate generation

### CODE REVIEW

File: src/main/scala/temple/generate/database/PostgresGenerator.scala [hunk 1]
```diff
@@ -74,6 +74,7 @@ object PostgresGenerator extends DatabaseGenerator[PostgresContext] {
       case DateCol       => s"DATE"
       case TimeCol       => s"TIME"
       case DateTimeTzCol => s"TIMESTAMPTZ"
+      case BlobCol       => s"BYTEA"
```
Comment (Lewiky) [inline src/main/scala/temple/generate/database/PostgresGenerator.scala]: 👋☕️

**Review by Lewiky — DISMISSED** [review]
Excellently done

File: src/it/scala/temple/generate/database/PostgresGeneratorIntegrationTest.scala [hunk 2]
```diff
@@ -57,6 +60,30 @@ class PostgresGeneratorIntegrationTest extends PostgresSpec with Matchers with B
     result.isLast shouldBe true
   }
 
+  // Since Blobs require files, we open create a test dynamically
+  it should "correctly store blobs" in {
+    val testImage  = new File("src/it/scala/temple/testfiles/cat.jpeg")
+    val fileStream = new FileInputStream(testImage)
+    try {
+      val fileContents    = fileStream.readAllBytes()
```
Comment (c-harding) [inline src/it/scala/temple/generate/database/PostgresGeneratorIntegrationTest.scala]: Can this be moved to a util please?

**Review by c-harding — DISMISSED** [review]
Looks good to go, though a util would be preferable to explicit file IO here

**Review by c-harding — APPROVED** [review]
thank you for your service in testing

**Review by Lewiky — APPROVED** [review]

**Pull request merged**